apps/mark-scan/frontend/src/pages/pat_device_identification/identify_input_step.tsx

```diff
@@ -37,6 +37,8 @@ export function IdentifyInputStep({
       event.preventDefault();
 
       if (event.key === behaviorToKeypressMap[inputName]) {
+        // Exhaustive switch statement so we don't need a default case
+        // eslint-disable-next-line default-case
```

Comment:
Generally our pattern is to have a default case with `throwIllegalValue(variableFromSwitch)` so that the type-checker enforces exhaustiveness.

Reply:
And to add a little more context: the reason this is different than simply suppressing the error is that using `throwIllegalValue` prevents forgetting to update `switch` statements when you add new possible values. Let's say you add a new possible value `'v'` to type `T` and you're not using `throwIllegalValue`. All of the `switch` statements that switch on a value of type `T` will still type check just fine. If you're using `throwIllegalValue` then its argument will have its type narrowed to `'v'` instead of `never`, resulting in a type error. This ensures you won't forget to update the relevant `switch` statements.